test/fixtures/dialects/snowflake/snowflake_select_union.yml

```diff
@@ -19,7 +19,7 @@ file:
           keyword: SELECT
           select_clause_element:
             literal: '2'
-    - orderby_clause:
-      - keyword: ORDER
-      - keyword: BY
-      - literal: '1'
+        unparsable:
+        - keyword: ORDER
+        - keyword: BY
+        - raw: '1'
```

Comment:
This looks wrong. But I just tried your branch and this works. Think you need to rerun the generate script to correct this:

```
python test/generate_parse_fixture_yml.py
```

Reply:
Yes! I re-ran the generator and this was corrected, so the file is no longer in the change